#car cmd: linear speed and steering angle


float32 linear_speed
float32 steering_angle_front
float32 steering_angle_rear
